src/renderer/components/dock/logs/__test__/download-logs.test.tsx

```diff
@@ -0,0 +1,186 @@
+/**
```

Comment:
This test suite is a behaviour, so lets move it to `behaviours`

Reply:
Moved into `behaviours`